scripts/marianasVariables.sh

```diff
@@ -1,68 +1,71 @@
-module load openblas-0.3.20-gcc-4.8.5-rpolrqa
-# openmpi@3.1.3%gcc@7.3.0~atomics~cuda~cxx~cxx_exceptions~gpfs~internal-hwloc~java~legacylaunchers~lustre~memchecker~pmi+romio+rsh~singularity+static+vt+wrapper-rpath fabrics=none schedulers=none arch=linux-centos7-x86_64_v3
-module load openmpi-3.1.3-gcc-7.3.0-2ocdjy4
-# openssl@1.1.1p%gcc@7.3.0~docs~shared certs=mozilla arch=linux-centos7-x86_64_v3
-module load openssl-1.1.1p-gcc-7.3.0-tz3ln5w
-# perl@5.26.0%gcc@7.3.0+cpanm+shared+threads patches=0eac10e,8cf4302 arch=linux-centos7-x86_64_v3
-module load perl-5.26.0-gcc-7.3.0-f7w3oxq
-# pkgconf@1.8.0%gcc@7.3.0 arch=linux-centos7-x86_64_v3
-module load pkgconf-1.8.0-gcc-7.3.0-gxowfdy
-# raja@0.14.0%gcc@7.3.0+cuda~examples~exercises~ipo+openmp~rocm+shared~tests build_type=RelWithDebInfo cuda_arch=60 arch=linux-centos7-x86_64_v3
-module load raja-0.14.0-gcc-7.3.0-vtbmo6k
-# suite-sparse@5.10.1%gcc@7.3.0~cuda~graphblas~openmp+pic~tbb arch=linux-centos7-x86_64_v3
-module load suite-sparse-5.10.1-gcc-7.3.0-thhoxwy
-# texinfo@6.5%gcc@7.3.0 patches=12f6edb,1732115 arch=linux-centos7-x86_64_v3
-module load texinfo-6.5-gcc-7.3.0-crm3bgr
-# umpire@6.0.0%gcc@7.3.0+c+cuda~device_alloc~deviceconst~examples~fortran~ipo~numa~openmp~rocm~shared build_type=RelWithDebInfo cuda_arch=60 tests=none arch=linux-centos7-x86_64_v3
-module load umpire-6.0.0-gcc-7.3.0-z22n3zy
-# zlib@1.2.12%gcc@7.3.0+optimize+pic+shared patches=0d38234 arch=linux-centos7-x86_64_v3
-module load zlib-1.2.12-gcc-7.3.0-hq7ha7b
+# autoconf@2.69%gcc@10.2.0 patches=35c4492,7793209,a49dd5b arch=linux-centos7-zen2
+module load autoconf-2.69-gcc-10.2.0-r677m42
+# autoconf-archive@2022.02.11%gcc@10.2.0 patches=130cd48 arch=linux-centos7-zen2
+module load autoconf-archive-2022.02.11-gcc-10.2.0-pbrbzut
+# automake@1.16.5%gcc@10.2.0 arch=linux-centos7-zen2
+module load automake-1.16.5-gcc-10.2.0-j4bwm4o
+# blt@0.4.1%gcc@10.2.0 arch=linux-centos7-zen2
+module load blt-0.4.1-gcc-10.2.0-tanugdw
+# ca-certificates-mozilla@2022-07-19%gcc@10.2.0 arch=linux-centos7-zen2
+module load ca-certificates-mozilla-2022-07-19-gcc-10.2.0-h2opehw
+# camp@0.2.3%gcc@10.2.0+cuda~ipo~rocm~tests build_type=RelWithDebInfo cuda_arch=60 arch=linux-centos7-zen2
+module load camp-0.2.3-gcc-10.2.0-vpkkybx
+# cmake@3.23.2%gcc@10.2.0~doc+ncurses+ownlibs~qt build_type=Release arch=linux-centos7-zen2
+module load cmake-3.23.2-gcc-10.2.0-i24avzq
+# coinhsl@2019.05.21%gcc@10.2.0+blas arch=linux-centos7-zen2
+module load coinhsl-2019.05.21-gcc-10.2.0-j7hsujd
+# cub@1.16.0%gcc@10.2.0 arch=linux-centos7-zen2
+module load cub-1.16.0-gcc-10.2.0-ovgrtom
+# diffutils@3.8%gcc@10.2.0 arch=linux-centos7-zen2
+module load diffutils-3.8-gcc-10.2.0-mjfwces
+# ginkgo@glu_experimental%gcc@10.2.0+cuda~develtools~full_optimizations~hwloc~ipo~oneapi+openmp~rocm+shared build_type=Release cuda_arch=60 arch=linux-centos7-zen2
+module load ginkgo-glu_experimental-gcc-10.2.0-dbmokiq
```

Comment:
@pelesh this is the version of ginkgo being used. If it's necessary, I can add more debugging information, or perhaps print the configuration used at the start of each pipeline?

Reply:
https://github.com/spack/spack/commit/68a1d558b146c6e98c16ca29eda596c5f231c065#diff-541117f56f263caf881b5f12084c0f41eda7f049a954c4590fd8282fa3b525e8 this is the spack commit that added this branch

Reply:
Can you run it manually on Marianas? The same test passes on other pipelines, so it looks as if this is Marianas specific issue.

Reply:
This is the [line](https://github.com/ginkgo-project/ginkgo/blob/glu_experimental/cuda/base/executor.cpp#L192) throwing the exception.

Reply:
Yes, this is confirmed failing on both P100 and V100 platforms on Marianas. I got the same error as seen in CI pipelines